h2/src/test/org/h2/test/synth/TestConcurrentUpdate.java

```diff
@@ -37,8 +36,11 @@ public static void main(String... a) throws Exception {
 
     @Override
     public void test() throws Exception {
+        if (!config.multiThreaded) {
+            return;
+        }
         deleteDb("concurrent");
-        final String url = getURL("concurrent;MULTI_THREADED=TRUE", true);
+        final String url = getURL("concurrent", true);
```

Comment:
BTW, this test looks a bit strange. `r.nextInt(3)` may return only 0, 1, or 2:
https://github.com/h2database/h2database/blob/5365e664b6560d757738919a7b5768e85e750890/h2/src/test/org/h2/test/synth/TestConcurrentUpdate.java#L66
But there is an additional unused block:
https://github.com/h2database/h2database/blob/5365e664b6560d757738919a7b5768e85e750890/h2/src/test/org/h2/test/synth/TestConcurrentUpdate.java#L81-L87